src/SampSharp.GameMode/World/Pickup.cs

```diff
@@ -47,9 +48,9 @@ public partial class Pickup : IdentifiedPool<Pickup>, IWorldObject
         /// <param name="position">The position where the pickup should be spawned.</param>
         /// <param name="virtualWorld">The virtual world ID of the pickup. Use -1 for all worlds.</param>
         /// <returns>The created pickup or null if it cannot be created.</returns>
-        public static Pickup Create(int model, int type, Vector3 position, int virtualWorld = -1)
+        public static Pickup Create(PickupModel model, PickupType type, Vector3 position, int virtualWorld = -1)
```

Comment:
Could you change this back to `(int model, int type, ...)` and then add overloads `(int model, PickupType type, ...)` and `PickupModel model, PickupType type)`?